debugging/book-library/script.js

```diff
@@ -29,17 +29,23 @@ const check = document.getElementById("check");
 //via Book function and start render function
 function submit() {
   if (
-    title.value == null ||
-    title.value == "" ||
-    pages.value == null ||
-    pages.value == ""
+    title.value === null ||
+    title.value === "" ||
+    author.value === null ||
+    author.value === "" ||
+    pages.value === null ||
+    pages.value === ""
```

Comment:
I'm still able to add books even if  there is only empty space entered in the form field. Please check submit  validation again
![image](https://github.com/user-attachments/assets/fa1b9057-59c6-4915-84d5-a6892a6940e4)